README.md

```diff
@@ -28,6 +28,12 @@ To install StagehandTesting, the animation snapshot testing utilities, add the f
 pod 'StagehandTesting'
 ```
 
+By default, this will use [SnapshotTesting](https://github.com/pointfreeco/swift-snapshot-testing) to record snapshots and perform comparisons. To instead use [iOSSnapshotTestCase](https://github.com/uber/ios-snapshot-test-case) as the snapshotting engine, set your test target dependency to use the `iOSSnapshotTestCase` subspec.
```

Comment:
Ultra-nit: might be slightly clearer to say "Point-Free's SnapshotTesting" and "Uber's iOSSnapshotTestCase"

Reply:
Yeah, fair enough 👍